src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -798,7 +798,10 @@ def setter(webapp):
 
 
 def show_identity(cmd, resource_group_name, name, slot=None):
-    return _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'get', slot).identity
+    op = _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'get', slot)
```

Comment:
NIT: make sure to use variable names that are self-descriptive Ex. here maybe call this app or web_app?